combiners/SSNet_Neural_Network.py

```diff
@@ -115,7 +116,7 @@ def nn(tr_list, imdb_tr_list, te_list, imdb_te_list):
 
     _acc = float(correct_pred)/float(TE_SAMPLE_SIZE)
     #print("Accuracy: ", float(correct_pred)/float(TE_SAMPLE_SIZE))
-    tr_acc = hist.history['acc'][EPOCH-1] * 100.
+    tr_acc = hist.history['accuracy'][EPOCH-1] * 100.
```

Comment:
Could you fetch dict items from hist.history and then use it.

Reply:
If I remember correctly, I updated this since the 'acc' attribute was not in `hist.history` for some reason; I might be using a different version of TF/Keras and forgot to take this out of the PR (also, if there is a specific version that should be used with these scripts, it might be worth including in the README/docs).